test/backend/PlotlyAPI.spec.js

```diff
@@ -24,31 +31,29 @@ describe('Grid API Functions', function () {
         // it works off of the assumption that a grid exists
 
         let fid;
-        return createGrid('Test updateGrid').then(res => res.json().then(json => {
-            // Update the grid's data
+        return createGrid('Test updateGrid')
+        .then(assertResponseStatus(201))
```

Comment:
Should 201 be defined as a comment?

Reply:
I've added a link to the documentation of Plot.ly's REST API